core/src/main/scala/org/http4s/internal/chunk.scala

```diff
@@ -0,0 +1,60 @@
+package org.http4s.internal
```

Comment:
This whole file can go away if and when https://github.com/functional-streams-for-scala/fs2/pull/1032 gets merged.